doc/api/http.md

```diff
@@ -496,6 +496,15 @@ added: v0.11.14
 If a request has been aborted, this value is the time when the request was
 aborted, in milliseconds since 1 January 1970 00:00:00 UTC.
 
+### request.connection
+<!-- YAML
+added: v0.3.0
+-->
+
+* `connection` {net.Socket}
```

Comment:
Nit: I think the name (`connection`) is not needed here as this is property not a function argument.